code/modules/mob/living/living.dm

```diff
@@ -1107,10 +1108,18 @@
 /mob/living/proc/get_visible_name()
 	return name
 
-/mob/living/update_gravity(gravity)
-	. = ..()
+/mob/living/proc/update_gravity(gravity)
+	// Handle movespeed stuff
+	var/speed_change = max(0, gravity - STANDARD_GRAVITY)
+	if(speed_change)
+		add_or_update_variable_movespeed_modifier(/datum/movespeed_modifier/gravity, multiplicative_slowdown=speed_change)
+	else
+		remove_movespeed_modifier(/datum/movespeed_modifier/gravity)
+
+	// If it's not started don't do anything
 	if(!SSticker.HasRoundStarted())
 		return
```

Comment:
this check feels awk where it is right now- if people (probably admins) are wandering around in space without gravity for whatever reason, why shouldn't they get gravity alerts? weird inconsistent stuff. should be scratched imo, save the time doing this check.

Reply:
I'm honestly not sure why this is here, I need to blame it tomorrow

Reply:
possibly alert throwing failing.

Reply:
circa https://github.com/tgstation/tgstation/pull/3885
will investigate more later

Reply:
I'm just gonna remove it on the basis that it's older then half our players. if it breaks then it should be pretty obvious and we can walk it back